VideoWeb/VideoWeb.Common/AppRoleService.cs

```diff
@@ -95,6 +94,11 @@ private static List<Claim> MapUserRoleToAppRole(List<JusticeUserRole> userRoles)
                 }
             }
             
+            if (userRoles.Exists(x => x == JusticeUserRole.VhTeamLead))
+            {
+                // Team leaders (Admins) are also VHOs so add the VHO role to get the same journey
+                claims.Add(new Claim(ClaimTypes.Role, AppRoles.VhOfficerRole));
```

Comment:
Worked this out with Lucy in a call. When a user only has Admin and nothing else they get 401 after log in because they don't fit any of the roles in the `navigator.component`. Currently only admin web cares about the difference between Admin and CSO. In Video web we mapped both roles to VHO. What we want for VHO is an admin to have the same journey as a CSO with a few extra privileges. It's easier to add the CSO role to provide the same journey than to go through the code and add `if role.contains(Admin)` in existing conditions. Let's have a call to go over it anyway.